package.json

```diff
@@ -106,14 +106,14 @@
     "gitbook-cli": "^2.3.2",
     "glob": "^7.1.6",
     "hard-source-webpack-plugin": "^0.13.1",
-    "http-proxy-middleware": "^1.0.5",
-    "jest": "^26.5.2",
+    "http-proxy-middleware": "^1.0.6",
+    "jest": "^26.5.3",
     "jest-canvas-mock": "^2.3.0",
     "jest-environment-jsdom": "^26.5.2",
     "jest-junit": "^12.0.0",
     "jest-serializer-vue": "^2.0.2",
-    "jstoxml": "^1.6.11",
-    "keep-a-changelog": "^0.10.1",
+    "jstoxml": "^2.0.1",
```

Comment:
@nicksellen maybe  this fixes the escaping?

Reply:
Yes, should do! Although I was hoping my test would fail when the upgrade happens, for trying to do double escaping... will investigate...